examples/jsm/loaders/FBXLoader.js

```diff
@@ -1298,6 +1299,35 @@ class FBXTreeParser {
 
 		}
 
+		// Sanitization: If geometry has groups, then it must match the provided material array.
+		// If not, we need to clean up the `group.materialIndex` properties inside the groups and point at a (new) default material.
+		// This isn't well defined; Unity creates default material, while Blender implicitly uses the previous material in the list.
+		if ( geometry.groups.length > 0 ) {
+
+			let needsDefaultMaterial = false;
+
+			for ( let i = 0, il = geometry.groups.length; i < il; i ++ ) {
+
+				const group = geometry.groups[ i ];
+
+				if ( group.materialIndex < 0 || group.materialIndex >= materials.length ) {
+
+					group.materialIndex = materials.length;
+					needsDefaultMaterial = true;
+
+				}
+
+			}
+
+			if ( needsDefaultMaterial ) {
+
+				const defaultMaterial = new MeshStandardMaterial();
```

Comment:
Can we use `MeshPhongMaterial` here? `FBXLoader` already uses `MeshPhongMaterial` as a default material in `parseMaterial()`.

Reply:
Fixed: https://github.com/mrdoob/three.js/commit/81dd060914058cc48aee7fe0d96deb1b6c84b63a

Reply:
Thanks for fixing! Our fork postprocesses all materials into MeshStandardMaterial, so that was a leftover.